samples/ipc_telemetry/producers/sensors_v1i1_interface.c

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+/********************************************************************
+ * This code was auto-generated from sensors v1 DL.
+ *
+ * Implement the code under the concrete functions.
```

Comment:
Think having something like the following under each function would be helpful:
```
static az_result sensors_1_subscribe_temperature_concrete(
    const az_ulib_telemetry_subscribe_model* const in,
    az_ulib_model_out* out)
{
  /* ***** User code here. ***** */
  (void)out;
 /* ***** End user code. ***** */
  return sensors_v1i1_subscribe_temperature(in->context, in->callback);
}
```

Reply:
Actually, this comment is not correct. Developers shall not change this file at all. The business logic shall be in the sensors_v1i1.c. I'll change the comment.

Reply:
Replaced comments by:
```
/********************************************************************
 * This code was auto-generated from sensors v1 DL and shall not be
 * modified.
 ********************************************************************/
```